src/programQueueProcessor/index.ts

```diff
@@ -111,6 +111,7 @@ const createProgramQueueProcessor = async ({
   return {
     knownEventTypes: {
       CLINICAL: KnownEventType.CLINICAL as KnownEventType.CLINICAL,
+      FILE: KnownEventType.FILE as KnownEventType.FILE,
       RDPC: KnownEventType.RDPC as KnownEventType.RDPC,
       SYNC: KnownEventType.SYNC as KnownEventType.SYNC,
```

Comment:
what does adding the `as KnownEventType.asdf` add here?

Reply:
I think it just enforces the type of the `knownEventTypes` object to be of type `{ FILE: KnownEventType.FILE, CLINICAL: KnownEventType.CLINICAL, ... }` rather than `{ FILE: KnownEventType, CLINICAL: KnownEventType,... }`

lol 
![image](https://user-images.githubusercontent.com/11821106/139929875-d60d38b8-e18f-4844-9b00-6e0801893b0b.png)

Reply:
I wonder if the same could have been done with `knownEventTypes: KnownEventType`, since typescript sometimes treats enums the same as actual objects... (eg you can do `Object.keys(SomeEnum)`

Reply:
knownEventTypes should have type `{[key: string]:KnownEventType}`

Reply:
which I expect would hold true without the extra `as KnownEventType.asdf` but I'm not inclined to go poking at this since it has little upside to cahnging it, just feels unnecessary.

Reply:
actually, just tested that, I believe the whole block can be replaced with `knownEventTypes: KnownEventType`.

Based on the following being valid: 
```ts
enum SomeEnum {
  key1 = "key1"
}

const someConst = SomeEnum
console.log(someConst.key1)
```

Reply:
yeah this makes sense. which raises the question: why not just export the enum?

Reply:
pretty sure `knownEventTypes` property was just there to provide access to the `KnownEventType` without requiring an extra import, ie it's just meant to be an alias... so `knownEventTypes: KnownEventType` should work. I wasn't aware of typescript's treatment of enums to be same as runtime object at the time, hence the verbose object

Reply:
*shurg* 

So long as the enum itself is also exported this is fine. Extra data is noisey but not harmful.

@Buwujiu you can listen to Minh's idea here and simplify the whole returned object if you think that will be easier to maintain.